<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejercicio 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1111</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Suma, Resta, Multiplicacion de Matrices</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz M</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_MatrizM">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_MatrizN">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>321</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_ResultadoMatriz">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>371</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_ColumnaM">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>340</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_FilaM">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_GenerarMatrizM">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar M</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_ColumnaN">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>330</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_FilaN">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_GenerarMatrizN">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_SumarMatriz">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>330</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Sumar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_RestarMatriz">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>380</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Restar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_MultiplicarMatriz">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>430</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiplicar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>472</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Girar Matriz</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_GeneraMatriz">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>331</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz N x N</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_tamanoMatriz">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>420</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese Tamaño</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_ResultadoGirarMatriz">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_GirarMatriz">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Girar Matriz</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_generarMatriz">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Encontrar Posicion</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_Fila">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_Columna">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="tw_matriz">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_GenerarMatrizM_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>440</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sb_buscarNumero">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>150</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese Numero Para Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bton_buscarNumero">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>200</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar Coordenada</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz Transpuesta</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablaT">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>480</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbColumnaT">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbFilaT">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>420</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Columna</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>420</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fila</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="transpuesta">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>90</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>430</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Matriz Transpuesta</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
